### PR TITLE
implementations for fake client pool constructor

### PR DESCRIPTION
**Pull request by yue9944882**
**What this PR does / why we need it**:

See #60390

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #60390

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: staging/src/k8s.io/client-go/dynamic/fake/client_pool.go [hunk 1]
```diff
@@ -21,10 +21,34 @@ package fake
...
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/testing"
 )
 
+// NewFakeClientPool returns an instance of FakeClientPool which respects manipulations
+// on resources and put them into storage.
+func NewFakeClientPool() *FakeClientPool {
+	o := testing.NewObjectTracker(scheme, codecs.UniversalDecoder())
```
Comment (yue9944882) [inline staging/src/k8s.io/client-go/dynamic/fake/client_pool.go]: For the reviewers, this constructor can help users to use fake `scheme` and fake `codecs`

**Comment by yue9944882** [comment]
flaky 
/test pull-kubernetes-e2e-kops-aws

**Review by yliaog — APPROVED** [review]

**Pull request closed without merging**